core/client/src/main/java/alluxio/client/block/RemoteBlockInStream.java

```diff
@@ -48,18 +48,17 @@
    *
    * @param blockId the block id
    * @param blockSize the block size
-   * @param workerNetAddress the address of the worker to read from
+   * @param address the worker address
    * @throws IOException if the block is not available on the remote worker
    */
-  public RemoteBlockInStream(long blockId, long blockSize, WorkerNetAddress workerNetAddress)
+  public RemoteBlockInStream(long blockId, long blockSize, WorkerNetAddress address)
```

Comment:
Any reason for changing the variable name? I think any of the variations is fine, but ideally we would want to be consistent.